estate/views/estate_property_views.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="estate_property_list" model="ir.ui.view">
+        <field name="name">estate.property.list</field>
+        <field name="model">estate.property</field>
+        <field name="arch" type="xml">
+            <list string="Estates" edit="false">
```

Comment:
Do you need edit="false" on here ? Isn't it the default when a form view exists ?